cmd/integrationArtifactUpload_test.go

```diff
@@ -0,0 +1,186 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+type integrationArtifactUploadMockUtils struct {
+	*mock.ExecMockRunner
+	*mock.FilesMock
+}
+
+func newIntegrationArtifactUploadTestsUtils() integrationArtifactUploadMockUtils {
+	utils := integrationArtifactUploadMockUtils{
+		ExecMockRunner: &mock.ExecMockRunner{},
+		FilesMock:      &mock.FilesMock{},
+	}
+	return utils
+}
+
+func TestRunIntegrationArtifactUpload(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Successfull Integration Flow Create Test", func(t *testing.T) {
+
+		tempDir, tmpErr := ioutil.TempDir("", "")
+		defer os.RemoveAll(tempDir) // clean up
+		assert.NoError(t, tmpErr, "Error when creating temp dir")
+		files := &mock.FilesMock{}
+		path := filepath.Join(tempDir, "iflow4.zip")
```

Comment:
Here we work with an absolute path. That must not be the case in real life. We have always workspace relative pathes. Reason for that is working with a temp folder instead of using the file mocking facilities. I would like to suggest to use the mock file system since it is closer to reality.